WcaOnRails/db/migrate/20160731000001_create_committees.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class CreateCommittees < ActiveRecord::Migration
+  def change
+    create_table :committees do |t|
+      t.string :name, limit: 50, null: false
+      t.string :slug, limit: 50, null: false
```

Comment:
I can't think of any good reason to have the SQL limit.

The validation is the right way to limit the size, and is easily changed if you want to.

There is some traditional belief that limiting the field saves disk space, but I don't think that is true in modern DBs, and even if it is, it would only matter if we had millions of Committees. And once we have that many committees, no work would get done anyway.